src/aio/gateway.rs

```diff
@@ -245,7 +244,11 @@ impl Gateway {
         let res = self
             .perform_request(
                 messages::DELETE_PORT_MAPPING_HEADER,
-                &messages::format_delete_port_message(protocol, external_port),
+                &messages::format_delete_port_message(
+                    self.control_schema.get("DeletePortMapping").unwrap(),
```

Comment:
Ditto?

Reply:
See my [previous comment](https://github.com/sbstp/rust-igd/pull/49#discussion_r422557050). I also made the same mistake in the synchronous version of Gateway and fixed it there, too.